documentation/Styles/general.css

```diff
@@ -29,4 +30,87 @@ ul,ol{
 button:hover{
     box-shadow: 5px 10px 5px var(--light-gray);
     cursor: pointer;
+}
+
+
+@import url("avatar/general.css");
+
+.nav{
+    display: grid;
+    grid-template-columns: 200px 1fr;
+    background-color: var(--header);
+    position: relative;
+    padding: 20px;
+    font-size: 22px;
+    border-radius: 0 0 15px 15px;
+    width: 100%;
+    position: fixed;
+    z-index: 1;
+}
+.nav-header{
+    font-size: 30px;
```

Comment:
suggestion: you may use rem here

Reply:
thank you I would definitely go through your suggestion.